src/led_matrix.py

```diff
@@ -1,18 +1,44 @@
-import cv2
+from os.path import exists
+import json
 import random
+import cv2
 import numpy as np
 
 from PIL import ImageEnhance, Image, ImageDraw, ImageFont
-from config import (
-    PIXEL_WIDTH,
-    PIXEL_HEIGHT,
-    BRIGHTNESS,
-    CONTRAST,
-    COLOR,
-    VIRTUAL_FRAMERATE,
-    PLAYLIST,
-    PLAYLIST_DELAY,
-)
+from .lib import colors
+
+CUSTOM_CONFIG = exists('config.json')
+CONFIG = 'default_config.json' if not CUSTOM_CONFIG else 'config.json'
+
+with open(CONFIG, mode='r',  encoding='utf8') as j_object:
+    cfg = json.load(j_object)
+
+# size of matrix
```

Comment:
I still need to move these notes to the README.md, and add a section on how to use the config.